python/hyperon/exts/snet_io/snet_gate.py

```diff
@@ -139,14 +143,58 @@ def get_service_messages(self):
     def get_operation_atom(self):
         return OperationAtom(self.service_details[1], self)
 
+    def __pretty_print_atoms__(self, input_atoms):
```

Comment:
I'd move this function out of `ServiceCall` and wouldn't introduce `len_threshold` and `current_len` as its fields.

Reply:
Okay, where to move this function? As i remember, you've proposed to move it to stdlib. Should i place it there?

Reply:
You can keep it in the same file for now. Just move it out of the class.

Reply:
@Necr0x0Der  Done.